lib/property/equatable.fz

```diff
@@ -40,3 +40,35 @@ public equatable is
   # as 'b'.
   #
   public type.equality(a, b equatable.this) bool => abstract
+
+
+  # is this part of given set
+  #
+  element_of(s container.Set equatable.this) => s.contains equatable.this
+
+
+  # is this not part of given set
+  #
+  not_element_of(s container.Set equatable.this) => !(equatable.this.element_of s)
+
+
+  # is `this` contained in `Set` `s`?
+  #
+  # This should usually be called using type inference as in
+  #
+  #   my_set := set_of ["A","B","C"]
+  #   say ("B" ∈ my_set)
+  #   say ("D" ∈ my_set)
+  #
+  public infix ∈ (s container.Set equatable.this) => element_of s
+
+
+  # is `this` not contained in `Set` `s`?
+  #
+  # This should usually be called using type inference as in
+  #
```

Comment:
Would be good to use triple backticks for code sections. When can then use this when generating docs.

Reply:
Let's use four space indentation in the future for code in comment, that should work:

    this is code

.